packages/react-reconciler/src/ReactFiberHooks.old.js

```diff
@@ -1784,20 +1786,35 @@ function warnOnOpaqueIdentifierAccessInDEV(fiber) {
     const name = getComponentName(fiber.type) || 'Unknown';
     if (getIsRendering() && !didWarnAboutUseOpaqueIdentifier[name]) {
       console.error(
-        'The object passed back from useOpaqueIdentifier is meant to be ' +
-          'passed through to attributes only. Do not read the ' +
-          'value directly.',
+        'Warning: The object passed back from useOpaqueIdentifier and those made from it are meant to be passed through to attributes only. Do not read the value directly.',
       );
       didWarnAboutUseOpaqueIdentifier[name] = true;
     }
   }
 }
 
+function warnOnDuplicateOpaqueIdentifierKeyInDEV(fiber) {
```

Comment:
Is this duplicated above?  If so, can we moved to a shared place?

Reply:
.old and .new seem to share everything except imports. I didn't want to make design decisions so I just did similarly to what was around me.